csv/readcsv.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"encoding/csv"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+
+	runtime "github.com/ahsanfayaz52/EmployeeManagementSystem"
+	"github.com/ahsanfayaz52/EmployeeManagementSystem/models"
+)
+
+func main() {
+	rt, err := runtime.NewRuntime()
+	if err != nil {
+		panic(err)
+	}
+	root, err := os.Getwd()
+	if err != nil {
+		log.Fatal(err)
+	}
+	filepath := "/Files/records.csv"
+	path := root + filepath
+	var rows = ReadCsv(path)
+	AddDataToDatabase(rows, rt)
+}
+
+// AddDataToDatabase adds employee data from csv to database.
+func AddDataToDatabase(rows [][]string, rt *runtime.Runtime) string {
+	for _, row := range rows {
+		var employee models.Employee
+		list := strings.Split(row[0], ",")
```

Comment:
we don't need to split it. `row[0]` should get the first column.

Reply:
done